include/deal.II/base/time_stepping.h

```diff
@@ -373,6 +399,123 @@ namespace TimeStepping
 
 
 
+  /**
+   * LowStorageRungeKutta is derived from RungeKutta and implement the explicit
+   * methods. The main advantages of low-storage methods are the reduced memory
+   * consumption and the reduced memory access.
+   */
+  template <typename VectorType>
+  class LowStorageRungeKutta : public RungeKutta<VectorType>
+  {
+  public:
+    using RungeKutta<VectorType>::evolve_one_time_step;
+
+    /**
+     * Default constructor. This constructor creates an object for which
+     * you will want to call <code>initialize(runge_kutta_method)</code>
+     * before it can be used.
+     */
+    LowStorageRungeKutta() = default;
+
+    /**
+     * Constructor. This function calls initialize(runge_kutta_method).
+     */
+    LowStorageRungeKutta(const runge_kutta_method method);
+
+    /**
+     * Initialize the explicit Runge-Kutta method.
+     */
+    void
+    initialize(const runge_kutta_method method) override;
+
+    /**
+     * This function is used to advance from time @p t to t+ @p delta_t. @p f
+     * is the function $ f(t,y) $ that should be integrated, the input
+     * parameters are the time t and the vector y and the output is value of f
+     * at this point. @p id_minus_tau_J_inverse is a function that computes $
+     * inv(I-\tau J)$ where $ I $ is the identity matrix, $ \tau $ is given,
+     * and $ J $ is the Jacobian $ \frac{\partial J}{\partial y} $. The input
+     * parameter are the time, $ \tau $, and a vector. The output is the value
+     * of function at this point. evolve_one_time_step returns the time at the
+     * end of the time step.
+     */
+    double
+    evolve_one_time_step(
+      const std::function<VectorType(const double, const VectorType &)> &f,
+      const std::function<
+        VectorType(const double, const double, const VectorType &)>
+        &         id_minus_tau_J_inverse,
+      double      t,
+      double      delta_t,
+      VectorType &y) override;
+
+    /**
+     * This function is used to advance from time @p t to t+ @p delta_t. This
+     * function is similar to the one derived from RungeKutta, but does not
+     * required id_minus_tau_J_inverse because it is not used for explicit
+     * methods. evolve_one_time_step returns the time at the end of the time
+     * step. Note that vec_ki holds the evaluation of the differential operator,
+     * and vec_ri holds the right-hand side for the differential operator
+     * application.
+     */
+    double
+    evolve_one_time_step(
+      const std::function<VectorType(const double, const VectorType &)> &f,
+      double                                                             t,
+      double      delta_t,
+      VectorType &solution,
+      VectorType &vec_ri,
+      VectorType &vec_ki);
+
+    /**
+     * Get the coefficients of the scheme.
```

Comment:
Would you mind expanding this a little bit to explain what the coefficients mean? `a` is not the conventional definition in terms of a Butcher tableau but merely one of the sub-diagonals. You might also want to refer to step-67 here for a more elaborate explanation.